src/app/screens/Publishers/index.tsx

```diff
@@ -90,21 +102,103 @@ function Publishers() {
 
   return (
     <Container>
-      <h2 className="mt-12 mb-2 text-2xl font-bold dark:text-white">
-        {t("used.title")}
-      </h2>
-
-      <p className="mb-6 text-gray-500 dark:text-neutral-500">
-        {t("used.description")}
-      </p>
-
-      {publishers.length > 0 ? (
-        <PublishersTable
-          publishers={publishers}
-          navigateToPublisher={navigateToPublisher}
-        />
-      ) : (
-        <p className="dark:text-white"> {t("used.no_info")}</p>
+      {(publishers.length > 0 || !hasTips) && (
```

Comment:
Does this mean a user needs to dismiss all tips first to see already used websites?

Reply:
This might just be irrelevant because we gonna handle this later in combination with #1905  
So no need to worry now, right @rolznz ? :)

Reply:
websites and `publishers` are the same thing, right?  Since it's an `||`, `publishers.length` (when they have websites/publishers) would trigger the block to show.

Reply:
Ah, thanks. Didn't read that in detail.